docs/operator.md

```diff
@@ -478,6 +478,22 @@ The [Kubernetes Pod tolerations  :octicons-link-external-16:](https://kubernetes
 | ----------- | ---------- |
 | :material-numeric-1-box: int         | `6000`     |
 
+### `replsets.primaryPreferTagSelector.region` 
+
+Makes MongoDB instance to be selected as Primary based on specified region
```

Comment:
I would reword it like: Ensures the MongoDB instance is selected as Primary based on specified region